backend/app/process_pdfs.py

```diff
@@ -0,0 +1,78 @@
+import os
+import camelot
+import fnmatch
+import pandas as pd
+
+#TO-DO: Check for PDFs resulting in empty table, edit tables to ensure data in formed correctly in CSV file
+# Idea: check for cells with \n characters using the tables' .data attribute and separate them into rows 
+# if each column has the same number of elements split by \n
+
+#Search through data folder & only collect PDF files
+def find_files(directory, pattern):
+    matches = []
+    for root, _, filenames in os.walk(directory):
+        for filename in fnmatch.filter(filenames, pattern):
+            matches.append(os.path.join(root, filename))
+    return matches  
+
+directory = 'data'
```

Comment:
It doesn't seem like we would benefit from this script using a relative 'data' path. On these grounds, I suggest changing this to the absolute path to the data folder (see my comment on line 65) so the script can also be run from the project root by: `python backend/app/process_pdfs.py`. Let me know if you think otherwise.